mqtt-transport-binding.md

```diff
@@ -0,0 +1,272 @@
+
+### 1.3. Content Modes
+
+The specification defines two content modes for transferring events:
+*structured* and *binary*.
+
+The *binary* mode *only* applies to MQTT 5.0, because of MQTT 3.1.1's lack of
+support for custom metadata.
+
+In the *structured* content mode, event metadata attributes and event data are
+placed into the MQTT PUBLISH message payload section
+using an [event format](#14-event-formats).
+
+In the *binary* content mode, the value of the event `data` attribute is placed
+into the MQTT PUBLISH message's payload section as-is, with
+the `contentType` attribute value declaring its media type; all other event
+attributes are mapped to the MQTT PUBLISH message's
+[properties section][5-publish-properties].
+
+### 1.4. Event Formats
+
+Event formats, used with the *stuctured* content mode, define how an event is
+expressed in a particular data format. All implementations of this
+specification MUST support the [JSON event format][JSON-format].
+
+MQTT 5.0 implementations MAY support any additional, including proprietary,
+formats.
+
+### 1.5. Security
+
+This specification does not introduce any new security features for MQTT, or
+mandate specific existing features to be used.
+
+## 2. Use of Cloud Events Attributes
+
+This specification does not further define any of the [Cloud Events][CE] event
+attributes.
+
+Two of the event attributes, `contentType` and `data` are handled specially
+and mapped onto MQTT constructs, all other attributes are transferred as
+metadata without further interpretation.
+
+This mapping is intentionally robust against changes, including the addition
+and removal of event attributes, and also accommodates vendor extensions to the
+event metadata. Any mention of event attributes other than `contentType` and
+`data` is exemplary.
+
+### 2.1. contentType Attribute
+
+The `contentType` attribute is assumed to contain a [RFC2046][RFC2046]
+compliant media-type expression.
+
+### 2.2. data Attribute
+
+The `data` attribute is assumed to contain opaque application data that is
+encoded as declared by the `contentType` attribute.
+
+An application is free to hold the information in any in-memory representation
+of its choosing, but as the value is transposed into MQTT as defined in this
+specification, the assumption is that the `data` attribute value is made
+available as a sequence of bytes.
+
+For instance, if the declared `contentType` is
+`application/json;charset=utf-8`, the expectation is that the `data` attribute
+value is made available as [UTF-8][RFC3629] encoded JSON text for use in
+AMQP.
+
+## 3. MQTT PUBLISH Message Mapping
+
+With MQTT 5.0, the content mode is chosen by the sender of the event. Gestures
+that might allow solicitation of events using a particular mode might be
+defined by an application, but are not defined here.
+
+The receiver of the event can distinguish between the two modes by inspecting
+the `content-type` message field. If the value is prefixed with the Cloud
```

Comment:
unclear what "value" is referring to. can you clarify.

Reply:
done